content/basics/ably.textile

```diff
@@ -0,0 +1,90 @@
+
+h2(#what). What can you build with Ably?
+
+Ably is used across a variety of industries to provide scalable building realtime collaborative web experiences, such as:
+
+* Live update services
+* Chat applications
+* Virtual events
+* Multi-user virtual spaces and applications
+* Internet of Things
+* And much more
+
+The following sections will cover some of the different APIs and features Ably provides which will give you a deeper insight into what can be done with Ably. There's also the "How to use Ably topic":/basics/use-ably which will go over the basics of connecting with Ably.
+
+Alternatively take a look at our "Use cases and solutions":https://ably.com/solutions topic for specific example use cases.
+
+h2(#about). About the APIs
+
+Ably supports both realtime and REST interfaces. The "Realtime library":/basics/use-ably#realtime-interface establishes a WebSocket connection allowing realtime messages to be efficiently sent both to and from the Ably. The "REST library":/basics/use-ably#rest-interface communicates using the HTTP protocol for all operations and is effectively stateless.
+
+There are a number of "Ably SDKs":https://ably.com/download in different programming languages to match your needs.
+
+Some of the key features are: 
+
+* "Pub/Sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). PubSub with Channels
+
+Ably uses the Publish/Subscribe (Pub/Sub) architectural pattern to allow any number of publishers to publish data and any number of subscribers to subscribe to a source of data. This works with Ably's concept of "channels". Channels are used to aggregate all data into a single location, allowing publishers to publish to just one location and subscribers to subscribe to just one source.
+
+Read more in our "Pub/Sub section":/channels.
+
+h3(#presence-and-occupancy). Presence and Occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events, such as entering, updating their state, or leaving the channel. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Read more in our "Presence and occupancy section":/presence-occupancy.
+
+h3(#message-storage-and-history). Message storage and History
```

Comment:
Inconsistent heading structure. Checking on our style guide we should be using sentence case. https://github.com/ably/docs/blob/main/writing-style-guide.md